gitlab/v4/cli.py

```diff
@@ -351,7 +355,7 @@ def extend_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         object_group = subparsers.add_parser(arg_name)
 
         object_subparsers = object_group.add_subparsers(
-            title="action", dest="whaction", help="Action to execute."
+            title="resource_action", dest="resource_action", help="Action to execute."
```

Comment:
The `title` is a user-facing human-readable string for the help so I think we could keep it, e.g. I think this is more consistent with other CLI prints:

```
usage: gitlab project-access-token [-h] {list,create,delete} ...

optional arguments:
  -h, --help            show this help message and exit

action:
  {list,create,delete}  Action to execute.
```

..than the new:
```
usage: gitlab project-access-token [-h] {list,create,delete} ...

optional arguments:
  -h, --help            show this help message and exit

resource_action:
  {list,create,delete}  Action to execute.
```
We could instead maybe extend the help string to say `Action to execute on the resource.`

Reply:
Thanks. Good point. Done.